client/routes/help/index.vue

```diff
@@ -75,33 +75,31 @@
           Cadence UI source code on GitHub
         </a>
       </div>
-
     </section>
     <section id="cli">
       <h3>
         <a href="#cli">
           Common CLI commands
         </a>
       </h3>
-      <p>Here are a some useful common CLI commands to get started with Cadence.</p>
-
-      <label for="cli-command-domain-register">Register a domain (local only)</label>
-      <pre id="cli-command-domain-register">cadence --domain {domain-name} domain register --global_domain false</pre>
-
-      <label for="cli-command-domain-describe">List domain settings</label>
-      <pre id="cli-command-domain-describe">cadence --domain {domain-name} domain describe</pre>
-
-      <label for="cli-command-workflow-run">Run a workflow</label>
-      <pre id="cli-command-workflow-run">cadence --domain {domain-name} workflow run --tl {task-list-name} --wt {workflow-type-name} --et 60 -i '"{input-string}"'</pre>
-
-      <label for="cli-command-workflow-show">See a workflow history</label>
-      <pre id="cli-command-workflow-show">cadence --domain {domain-name} workflow show -w {workflow-id} -r {run-id}</pre>
-
-      <label for="cli-command-workflow-list">List closed workflows</label>
-      <pre id="cli-command-workflow-list">cadence --domain {domain-name} workflow list</pre>
+      <p>
+        Here are a some useful common CLI commands to get started with Cadence.
+      </p>
 
-      <label for="cli-command-workflow-list-open">List open workflows</label>
-      <pre id="cli-command-workflow-list-open">cadence --domain {domain-name} workflow list --open</pre>
+      <section v-for="commandGroup in cliCommands" :key="commandGroup.header">
+        <h5>{{ commandGroup.header }}</h5>
+        <div
+          class="cli-command"
+          v-for="command in commandGroup.commands"
+          :key="command.id"
+        >
+          <label :for="command.id">{{ command.label }}</label>
+          <pre :id="command.id">{{ command.value }}</pre>
+          <p v-for="(description, index) in command.describe" :key="index">
```

Comment:
Maybe call this `command.description` instead of `command.describe`?

Reply:
yea I was thinking that also. I'll update